components/script/dom/html/htmliframeelement.rs

```diff
@@ -730,21 +721,15 @@ impl VirtualMethods for HTMLIFrameElement {
             .attribute_mutated(attr, mutation, can_gc);
         match *attr.local_name() {
             local_name!("sandbox") => {
-                self.sandbox_allowance
+                self.sandbox.set(None);
```

Comment:
```suggestion
                // When an iframe element's sandbox attribute is removed while it has
                // a non-null content navigable, the user agent must empty the iframe
                // element's iframe sandboxing flag set.
                //
                // TODO
                self.sandbox.set(None);
```

Reply:
I've added this, but I'm not sure about the `TODO` as I think this code also clears the flag set as well appropriately.